homeassistant/components/devolo_home_control/climate.py

```diff
@@ -107,6 +110,6 @@ def temperature_unit(self) -> str:
     def set_hvac_mode(self, hvac_mode: str) -> None:
         """Do nothing as devolo devices do not support changing the hvac mode."""
 
-    def set_temperature(self, **kwargs):
+    def set_temperature(self, **kwargs: int) -> None:
```

Comment:
kwargs can be a lot, but not an int 😉 
```suggestion
    def set_temperature(self, **kwargs: Any) -> None:
```

Reply:
True, the type hints of vscode confused me. Should I make it Any or if I know I can handle just certain types make it e.g. dict[str, int]?

Reply:
Any is fine here :)